### DISCUSSION TITLE
Still maintained?

### DISCUSSION THREAD

**Discussion by sylvainbx** [Jimskapt/rust-book-fr#400] [General]
Is this repository still maintained or abandoned?

**Comment by EWE-2** [comment]
Is the repo still maintened ? Can contribute ?

**Reply by LouGit** [reply]
Yes, it is still maintained, I can tell because one of my PR was accepted a couple of weeks ago or so.

I'm now quite busy updating the French translation by comparing the old English version (upon which the French translation was based) to the current English version, so that, eventually, the French translation matches the current version of the Book.
